File: switch_case/switch_case.py
# use switch if there are more than 3 options,
# otherwise you can use if / elif / else

def chosen_option(opt):
   options = {
      1: "you have won a car",
      2: "you have won a coffee mug",
      3: "you have to pay a fee"
   }
   return options.get(opt, "non-existing option chosen. you lose!")


 
x =int(input ("choose option 1, 2, or 3:  "))   # note y by default is of type
                                                # string and must be converted
                                                # to int to match the dict key.
resp = chosen_option(x)
print(resp)

File: 4.switch_case/switch_case.cpp

// switch makes sense when the number of choices is larger than 2
// other wise can use if / else if

#include<iostream>

int main() {

int x ;

std::cout << " door #1\n";
std::cout << " door #2\n";
std::cout << " door #3\n";

std::cout << " choose one option : ";
std::cin >> x;

switch (x)
   {
      case 1:
         std::cout << " you have won a car";
         break;

      case 2:
         std::cout << " you have won a coffee mug";
         break;

      case 3:
         std::cout << " you have to pay a fee ";
         break;

      default:
         std::cout << " you entered a non-existing option; you lose!";
         break;


   }

}
